index.html

```diff
@@ -5,184 +5,285 @@
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="button">
+                    <button type="button" class="work-button">See Project</button>
+                </div>
+            </div>
+        </div>
+        <div class="grid-item grid-item4">
+            <div class="work-img work-img4">
+
+            </div>
+            <div class="description-container4">
+                <h2>Tonic</h2>
+                <div class="work-content work-content1">
+                    <p>CANCOPY</p>
+                    <ul>
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div class="work-description">
+                    <p class="p-1">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                </div>
+                <div class="categories">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="button">
+                    <button type="button" class="work-button">See Project</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section id="about" class="about-myself">
+        <div class="about-description">
+            <div class="about-header">
+                <h2>About <br> Myself</h2>
             </div>
-            <div>
-                <a href="">
-                    <img src="./img/Vector.png" alt="twitter logo">
-                </a>
+            <div class="about-content">
+                <p class="about-content-p">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <h4 class="connect">LET'S CONNECT</h4>
+                <ul>
+                    <li>
+                        <a href="twitter.com"><img src="./img/Vector.png" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="medium.com"><img src="./img/Vector2.png" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="linkedin.com"><img src="./img/Linkedin-icon.png" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="github.com"><img src="./img/Vector3.png" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="angellist.com"><img src="./img/Vector4.png" alt=""></a>
+                    </li>
+                </ul>
+                <button type="button" class="my-button">Get My Resume</button>
```

Comment:
- [ ] Kindly make sure the **About me section** design sticks to the one from Figma design and also remember the add the **Chevron buttons as from Figma design** e.g: **`>`** and use the correct text color to make you project sticks to the Figma design.
![aaa](https://user-images.githubusercontent.com/83042742/173172702-5961d3f1-eafd-4a64-8799-2ffd92c16983.png)

### Figma design
![ab_fi](https://user-images.githubusercontent.com/83042742/173172717-2b1518ad-2c11-4a48-a254-60f333cbada7.png)
### Figma design
![abou_s](https://user-images.githubusercontent.com/83042742/173172735-4801d463-230f-4fe0-9c9b-0f84c478dfe6.png)